src/Components/ADT3DSceneBuilder/Internal/Shared/SharedFormUtils.tsx

```diff
@@ -0,0 +1,39 @@
+import { Icon, IDropdownOption } from '@fluentui/react';
+import React from 'react';
+
+const iconStyles = { marginRight: '8px' };
+const optionWrapperStyle = { display: 'flex', alignItems: 'center' };
+
+export const onRenderTypeOption = (option: IDropdownOption): JSX.Element => {
```

Comment:
might be nice to have function comments here about when this is used just so you don't have to look up where it's referenced.

Reply:
And what it's for. If I recall, this is to put an icon next to each entry in the dropdown menu. So the data field needs to be set on the option object.